Source/Scene/Gltf3DTileContent.js

```diff
@@ -176,8 +177,12 @@ Gltf3DTileContent.prototype.hasProperty = function (batchId, name) {
   return false;
 };
 
-Gltf3DTileContent.prototype.getFeature = function (batchId) {
-  return undefined;
+Gltf3DTileContent.prototype.getFeature = function (featureId) {
+  return this._model._featureTable.getFeature(featureId);
+};
+
+Gltf3DTileContent.prototype.getProperty = function (featureId, name) {
+  return this._model._featureTable.getProperty(featureId, name);
```

Comment:
Should ModelExperimental have a getter for `featureTable`? just mark it as private API?

See `Cesium3DTileContent#batchTable` for example.